### PR TITLE
ex15_ch4.py

### PR DESCRIPTION
**Pull request by amymhaddad**
Instructions: Create a simple program that validates user login credentials. The program must prompt the user for a username and password. The program should compare the password given by the user to a known password. If the password matches, the program should display “Welcome!” If it doesn’t match, the program should display “I don’t know you.”

Constraints:
• Use an if/else statement for this problem. • Make sure the program is case sensitive.

### CODE REVIEW

File: ex15_ch4.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: create a constant
+KNOWN_PASSWORD = "secret"
```
Comment (paulghaddad) [inline ex15_ch4.py]: Excellent!

File: ex15_ch4.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+#step2: get input on user's username
+username = input("Enter your username: ")
+
+#step3: get input on user's password
+password = input("Enter your password: ")
+
+#step4: write if statement to verify user password with known password
+if password == KNOWN_PASSWORD:
```
Comment (paulghaddad) [inline ex15_ch4.py]: Excellent, this is perfect! 👏

File: ex15_ch4.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    print("Welcome")
+
```
Comment (paulghaddad) [inline ex15_ch4.py]: No blank line needed between if, elif or else statements.

**Comment by amymhaddad** [comment]
@paulghaddad deleted space between "if" and "else". Please approve!

**Pull request merged**